src/styles/layout/index.md.njk

```diff
@@ -123,3 +123,18 @@ The desktop specific classes also allow you to set the width of the tablet break
 ### Nested grids
 
 {{ example({group: "styles", item: "layout", example: "nested", html: true, open: true, size: "m"}) }}
+
+## Override how elements display
+
+You can use display override classes if you need to override how elements display on the page.
+
+Use:
+
+- `govuk-!-display-block` to display as a block
+- `govuk-!-display-inline` to display inline
+- `govuk-!-display-inline-block` to display as an inline block
+- `govuk-!-display-none` to remove the element from the page
+
+You can also remove elements from the printed version of the page using `.govuk-!-display-none-print`.
```

Comment:
Could we also remove the `.` from this class name so it becomes `govuk-!-display-none-print`.